File: Domain/Entities/Room.cs
using Domain.Base;

namespace Domain.Entities
{
    public class Room : IEntityBase
    {
        public Guid Id { get; set; }
        public int Beds { get; set; }
        public int BedsAvailable { get; set; }
        public int RoomNumber { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public ICollection<Patient> Patients { get; set; }
        public Department Department { get; set; }
        public Guid DepartmentId { get; set; }

    }
}

File: client/src/app/models/Room.ts
import Patient from "./Patient";

export default interface Room {
    id: string;
    beds: string ;
    bedsAvailable: string;
    //isFree: boolean;
    createdAt: Date;
    updatedAt: Date;
    roomNumber: string ;
    patients: Patient[];
    departmentId: string;
}